src/main/java/org/tl/Hobbies.java

```diff
@@ -1,68 +1,183 @@
 package org.tl;
 
-import java.util.List;
-import java.util.Map;
+import java.io.*;
+import java.util.*;
 
 public class Hobbies {
 
+
     /**
      * Collect lines from the given file into a list.
      *
      * @param filePath The path to the file to read.
      * @return A list of lines from the file.
      */
     public List<String> createListFromFile(String filePath) {
-        return null;
+        List<String> lines = new ArrayList<>();
+        try {
+            File file = new File(filePath);
+            Scanner scanner = new Scanner(file);
+            while (scanner.hasNextLine()) {
+                String line = scanner.nextLine();
+                lines.add(line);
+            }
+
+            scanner.close();
+        } catch (FileNotFoundException e) {
+            System.out.printf("Error occurred %s%n", e.getMessage());
+        }
+        return lines;
     }
 
+
     /**
      * Create a dictionary of people's hobbies from the given file.
      *
      * @param filePath The path to the file to read.
      * @return A map where each person's name is associated with a list of their hobbies.
      */
     public Map<String, List<String>> createDictionary(String filePath) {
-        return null;
+        Map<String, List<String>> result = new HashMap<>();
+
+        try {
+            FileReader fileReader = new FileReader(filePath);
+            Scanner scanner = new Scanner(fileReader);
+            createListFromFile(filePath);
+                String line = scanner.nextLine();
+                String[] parts = line.split(":");
+                String name = parts[0].trim();
+                String[] hobbies = parts[1].split(",");
+                List<String> hobbyList = new ArrayList<>(Arrays.asList(hobbies));
+                result.put(name,hobbyList);
+
+            scanner.close();
```

Comment:
you don't need filereader here as well as scanner.
Use the result of a `createListFromFile(filePath);` 
Then iterate over it in a loop and put correct things to the list. 
Right not it puts to the map on the first person's hobby. 
Also, each line contains one row, so 
`String[] hobbies = parts[1].split(",");` has only one element anyway